### ISSUE TITLE
Konfiguration eins PIDs kann nicht gespeichert werden

### ISSUE THREAD

**Issue by nemodil** [mcm4iob/ioBroker.pid#274]
Hallo,
Die Konfiguration meines bestehenden bzw. eines neuen Reglers kann nicht mehr gespeichert werden.
Mir ist dies jetzt erst aufgefallen, da ich einen neuen Regler definieren wollte.
Bei einer neuen Instanz1 besteht das Problem auch, die ID ist rot sowie auch Save und Save and Close.
Oder mach ich etwas falsch...?

Meine System:
Platform: x64 32GB
OS: Linux
Node.js: v20.18.1
NPM: 10.8.2
js-controller: 7.0.6

![PID Config](https://github.com/user-attachments/assets/9f2ba87b-09f0-451b-917d-ab6255b92c05)

**Comment by mcm1957** [comment]
Which admin version do you use?

**Comment by nemodil** [comment]
Currently I use admin version 7.4.8.

**Comment by mcm1957** [comment]
Please use stable version of admin to change config until admin bug is fixed.

**Comment by nemodil** [comment]
Okay, thanks!

**Comment by mcm1957** [comment]
Please check with admin 7.4.9

**Comment by nemodil** [comment]
Yes! With Admin 7.4.9, I can save my PID settings again. Thanks!

**Comment by mcm1957** [comment]
TODO:
Block admin >7.1.5 and <7.4.9
